perf/benchmark/runner/runner.py

```diff
@@ -22,13 +22,17 @@
 import shlex
 import uuid
 import sys
+import tempfile
+import time
 from subprocess import getoutput
 from urllib.parse import urlparse
 import yaml
 from fortio import METRICS_START_SKIP_DURATION, METRICS_END_SKIP_DURATION
 
-
+NAMESPACE = os.environ.get("NAMESPACE", "twopods")
+NIGHTHAWK_GRPC_SERVICE_PORT_FORWARD = 9999
 POD = collections.namedtuple('Pod', ['name', 'namespace', 'ip', 'labels'])
+NIGHTHAWK_DOCKER_IMAGE = "envoyproxy/nighthawk-dev:latest"
```

Comment:
should be pinned to a specific version or the benchmark results may change over time

Reply:
good catch! yes, we should. I need to sync with Otto about this.

Reply:
@oschaaf I'm doing some refactor to get your Nighthawk PR merged in, which version do you think we can use here?

Reply:
@carolynhu for now /the short term, maybe pin to the latest revision:
envoyproxy/nighthawk-dev:59683b759eb8f8bd8cce282795c08f9e2b3313d4
I'll create a PR to change that once we have a version tag for this that we can use instead.

Reply:
yeah, sure